docs/sdk/examples/assets/pastefromoffice/pastefromoffice.js

```diff
@@ -0,0 +1,164 @@
+var sources = [
+    {
+        name: 'Google Docs',
+        link: 'pastefromgoogledocs.html',
+        linkName: 'Paste from Google Docs',
+        regex: /id=(\"|\')?docs\-internal\-guid\-/,
+        isAlreadyShow: false
+    },
+    {
+        name: 'Microsoft Word',
+        link: 'pastefromword.html',
+        linkName: 'Paste from Word',
+        regex: getProperWordRegex(),
+        isAlreadyShow: false
+    },
+    {
+        name: 'Microsoft Excel',
+        link: 'pastefromexcel.html',
+        linkName: 'Paste from Excel',
+        regex: /<meta\s*name=generator\s*content="Microsoft\s*Excel\s*\d+"?\/?/gmi,
+        isAlreadyShow: false
+    },
+    {
+        name: 'LibreOffice',
+        link: 'pastefromlibreoffice.html',
+        linkName: 'Paste from LibreOffice',
+        regex: /<meta\s+name=["']?generator["']?\s+content="LibreOffice\s*\d+"?\/?/gmi,
+        isAlreadyShow: false
+    }
+];
+var instances = getInstances();
+// A state variable that indicates if the clipboard notification has been seen.
+// We use the variable for displaying the notification only once per demo.
+let hasNotificationBeenSeen = false;
```

Comment:
Mixed `var` and `let` keywords. Let's stick to `var` as this JS is compiled to support older browsers.